src/tools/miri/cargo-miri/src/phases.rs

```diff
@@ -501,6 +501,10 @@ pub fn phase_runner(mut binary_args: impl Iterator<Item = String>, phase: Runner
     // Set missing env vars. We prefer build-time env vars over run-time ones; see
     // <https://github.com/rust-lang/miri/issues/1661> for the kind of issue that fixes.
     for (name, val) in info.env {
+        // See <https://github.com/rust-lang/   /pull/   >.
```

Comment:
This needs to be filled in before merge.

Reply:
Sure.
This pr have a lot of things to be cleaned before merge now.